packages/manager/modules/pci/src/projects/project/project.constants.js

```diff
@@ -78,40 +78,52 @@ export const DOCUMENTATION_LINKS = [
     term: 'pci_projects_project_documentation_getting_started_term',
     description:
       'pci_projects_project_documentation_getting_started_description',
-    href:
+    asiahref:
+      'https://docs.ovh.com/asia/en/public-cloud/public-cloud-essential-information/',
+    defaulthref:
```

Comment:
**question:** If we are displaying same content for all other subsidiary, why change it only for India?

Reply:
Hi Anoop, Will check with Marion regarding this.